calendar-bundle/src/Resources/contao/modules/ModuleEventMenu.php

```diff
@@ -106,6 +106,11 @@ protected function compileYearlyMenu()
 		{
 			foreach ($arrDay as $arrEvents)
 			{
+				if (!isset($arrData[substr($intDay, 0, 4)]))
+				{
+					$arrData[substr($intDay, 0, 4)] = 0;
+				}
+
 				$arrData[substr($intDay, 0, 4)] += \count($arrEvents);
```

Comment:
What about a variable for `$arrData[substr($intDay, 0, 4)]`? This would also make clear what this does.

Reply:
See #2583